docs/developer-guide/scaffold.md

```diff
@@ -0,0 +1,38 @@
+---
+order: 3
+description: How to scaffold a blockchain.
+---
+
+# Scaffold a Blockchain
+
+The project directory of any Cosmos SDK blockchain contains many directories, source code files, configuration files, scripts, etc. Some of these files implement custom logic and are very specific to a particular project. Other files, however, are common between different Cosmos SDK projects and act as wiring between different parts of the project. Starport CLI automatically generates this common (boilerplate) code and helps in scaffolding custom functionality, so that developers can focus on application-specific logic.
```

Comment:
```suggestion
The project directory of any Cosmos SDK blockchain contains many directories, source code files, configuration files, scripts, and so on. Some of these files implement custom logic and are very specific to a particular project. Other files, however, are common between different Cosmos SDK projects and act as wiring between different parts of the project. 

The common boilerplate code is automatically generated by Starport CLI. This code helps in scaffolding custom functionality so that developers can focus on building application-specific logic.
```

Reply:
let's not use latin